src/Coordination/KeeperServer.h

```diff
@@ -41,13 +46,16 @@ class KeeperServer
 
     /// Almost copy-paste from nuraft::launcher, but with separated server init and start
     /// Allows to avoid race conditions.
-    void launchRaftServer(
-        bool enable_ipv6,
-        const nuraft::raft_params & params,
-        const nuraft::asio_service::options & asio_opts);
+    void launchRaftServer(bool enable_ipv6);
 
     void shutdownRaftServer();
 
+    void loadLatestConfig();
+
+    void recoveryMode(nuraft::raft_params & params);
```

Comment:
The semantics of this function are unclear from its name. `isRecoveryMode`, `setRecoveryMode`, `enterRecoverMode`?

Reply:
`enterRecoverMode` is good. 😄